channels/ca.m3u

```diff
@@ -210,6 +214,8 @@ http://primeasia.dyndns.tv:8080/Live_web_250/index.m3u8
 http://cdn27.live247stream.com/primecanada/247/primecanada/stream1/playlist.m3u8
 #EXTINF:-1 tvg-id="QVTV.ca" tvg-country="CA" tvg-language="English" tvg-logo="https://i.imgur.com/X9LBFzK.jpg" group-title="Religious",QVTV (720p)
 https://qvmstream.tulix.tv/720p/720p/playlist.m3u8
+#EXTINF:-1 tvg-id="SikhSpiritualCentreRexdale.com" tvg-country="CA" tvg-language="Punjabi" tvg-logo="https://www.sikhspiritualcentrerexdale.com/wp-content/uploads/2019/07/favicon-transprent.png" group-title="",Sikh Spiritual Centre Rexdale
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="SikhSpiritualCentreRexdale.ca" tvg-country="CA" tvg-language="Punjabi" tvg-logo="https://www.sikhspiritualcentrerexdale.com/wp-content/uploads/2019/07/favicon-transprent.png" group-title="",Sikh Spiritual Centre Rexdale
```

Reply:
@freearhey Can we use site address in the tvg-id? Some channels are located at standard `.com` address and do not end with `.ca`.

Reply:
tvg-id isn't the address of the website. It's an unique name that permits to identify a channel.